promql/value.go

```diff
@@ -267,7 +282,7 @@ func (m Matrix) String() string {
 func (m Matrix) TotalSamples() int {
```

Comment:
The doc comment here should be updated to reflect that histograms are taken into account in a weighted fashion.